hash.go

```diff
@@ -382,3 +384,30 @@ func (db *RoseDB) HIncrBy(key, field []byte, incr int64) (int64, error) {
 	}
 	return valInt64, nil
 }
+
+// GetHashKeys get all stored keys of type Hash.
+func (db *RoseDB) GetHashKeys() (keys [][]byte, err error) {
+
+	db.hashIndex.mu.RLock()
+	defer db.hashIndex.mu.RUnlock()
+
+	for key, idxTree := range db.hashIndex.trees {
+
+		rowValue := idxTree.Get([]byte(key))
+		if rowValue == nil {
+			continue
+		}
+
+		idxNode, _ := rowValue.(*indexNode)
+		if idxNode == nil {
+			continue
+		}
+
+		ts := time.Now().Unix()
```

Comment:
Do we need to get the current time before the for loop? What do you think? Do we consider the time that we call the function? I think, to make it more consistent, we can call this function just before the for loop. I am open to listen to reasons if I miss something.

Reply:
In my opinion, if there are huge of keys, maybe the ts will be delay 1 millisecond, and the result maybe not the same.

Reply:
I mean if we get the current timestamp on for loop, there might be some inconsistent situations. For example, in the hash, as I know, there is no exact order which means that the keys can be retrieved in any order. Also, I think, a more important part is that when we call the command, we expect to check to expire time in one exact time. As a user, I don't want the key to depend on the for loop or internal logic of the function. So, I think we can get the current timestamp just before the for loop. Please let me know if I miss something.

Reply:
I see, I knew what's your point, I will update it.